.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+Documentation:
+  Enabled: false
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
Any chance we can stick with single quotes? Only speaking for myself here, the reasons why this is important: 

* The difference adds a semantic benefit when scanning code, to quickly recognize what strings are interpolated. 
* The choice of single vs. double conveys intention.
* Editors can pick up on the difference and syntax highlight accordingly. 
* Provides an additional matching pattern “hook”, upon which to grep. 

This is obviously my own preference, and I definitely understand not all folks benefit from such things. Thoughts?